S3 data lake Glue partition permissions (#1161)
rudderlabs/rudderstack-docs: docs/destinations/warehouse-destinations/s3-datalake.mdx

<li>glue:CreateDatabase</li>
<li>glue:GetTables</li>
</ul>
<br />

> If you have enabled <a href="https://docs.aws.amazon.com/glue/latest/dg/aws-glue-api-catalog-partitions.html">catalog partitions</a> on AWS Glue, you need the following additional permissions:

Author: ```suggestion
The following additional permissions are required to enable the <a href="https://docs.aws.amazon.com/glue/latest/dg/aws-glue-api-catalog-partitions.html">AWS Glue partitions</a>:
```
